__tests__/utils.tsx

```diff
@@ -214,30 +204,27 @@ describe('getPathInfo()', () => {
       lang = Instance.En
     ): Promise<MockedRequest> {
       let request!: MockedRequest
-      mockHttpPost(apiEndpoint, (req, res, ctx) => {
-        request = req
-
-        return returnApiUuid({
-          __typename: 'Article',
-          alias: path,
-        })(req, res, ctx)
-      })
+      global.server.use(
+        rest.post(apiEndpoint, (_req, res, ctx) => {
+          return res(ctx.json({ data: {  __typename: 'Article',
+          alias: path } }))
+        })
+      )
 
       await getPathInfo(lang, path)
 
       return request
     }
-  })
 
   describe('returns null', () => {
     test('when there was an error with the api call', async () => {
-      mockHttpPost(apiEndpoint, returnText('', { status: 403 }))
+      mockApi(apiEndpoint, returnText('', { status: 403 }))
 
       expect(await getPathInfo(Instance.En, '/path')).toBeNull()
     })
 
     test('when api response is malformed JSON', async () => {
-      mockHttpPost(apiEndpoint, returnText('malform json'))
+      mockApi(apiEndpoint, returnText('malform json'))
```

Comment:
This will not work. Add a function `apiToReturnMalformedJson()` similar to `apiToReturnError()`. This function should be written in a way that it returns malformed json